app/controllers/ride_mechanics_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RideMechanicsController < ApplicationController
+  def create
```

Comment:
This works great (and your controller set up looks good) but because you're just doing a redirect at the end, you don't actually need to save any of these variables as instance variables. You're not rendering a view from this action so you could just do regular ol' variables without the @